Tools/default.editorconfig

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) Microsoft. All rights reserved.
```

Comment:
can you copy the editorconfig from sarif-pattern-matcher?

Reply:
nice, done.